librosa/core/spectrum.py

```diff
@@ -229,6 +229,7 @@ def stft(y, n_fft=2048, hop_length=None, win_length=None, window='hann',
     # how many columns can we fit within MAX_MEM_BLOCK?
     n_columns = int(util.MAX_MEM_BLOCK / (stft_matrix.shape[0] *
                                           stft_matrix.itemsize))
+    n_columns = max(n_columns, 1)
```

Comment:
LGTM.  Now might also be a good opportunity to switch this over to use integer division `//` and drop the explicit int cast to be more python3ic.